RuleDocumentation/AvoidUnusableParameter.md

```diff
@@ -0,0 +1,44 @@
+# AvoidUnusableParameter
+
+**Severity Level: Warning**
+
+## Description
+
+Parameter name should follow good naming practices. If parameter name cannot be parsed properly, then it can not be used in a standard way.
```

Comment:
maybe just re-word it that in certain scenarios, parsing can fail.

Reply:
Done